pypeit/io.py

```diff
@@ -749,7 +749,12 @@ def fits_open(filename, **kwargs):
     Raises:
         PypeItError: Raised if the file does not exist.
     """
-    if not os.path.isfile(filename):
+    # TODO: Need to allow for filename to be an _io.FileIO object for use with
```

Comment:
i think `pathlib` provides more sophisticated tools for checking files types/status. might be worth a look.

Reply:
I tried just instantiating a `Path` object from the `_io.FileIO` object, and that failed.  After a quick Google search, I couldn't find an obvious solution.  Leaving it (and the TODO comment) as is, for now.